gemfiles/rails_6.0.0.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rails",   "~> 6.0.0"
+gem "sqlite3", "~> 1.4.1"
```

Comment:
I understand you're doing this to fix the issue that Rails 6 requires `sqlite` version `>= 1.4`. A better approach would be to keep the development dependency set on the `gemspec` and only override the version where needed directly on the `Appraisals` file.

Supposing the `sqlite` version is kept at the current version on the `gemspec`, we could do this on the `Appraisals` for example:

```ruby
appraise 'rails_6.0.0' do
  gem 'rails', '~> 6.0.0'
  # Rails 6 requires a newer version of the sqlite3 gem
  gem 'sqlite3', '~> 1.4.0'
end

%w[5.2.0 5.1.0 5.0.0 4.2.0].each do |rails_version|
  appraise "rails_#{rails_version}" do
    gem 'rails', "~> #{rails_version}"
  end
end
```
Another approach would be to update the `sqlite3` version on the `gemspec` to `~> 1.4.0` and then have this `Appraisals` file instead:
```ruby
%w[6.0.0 5.2.0 5.1.0].each do |rails_version|
  appraise "rails_#{rails_version}" do
    gem 'rails', "~> #{rails_version}"
  end
end

%w[5.0.0 4.2.0].each do |rails_version|
  appraise "rails_#{rails_version}" do
    gem 'rails', "~> #{rails_version}"
    gem 'sqlite3', "~> 1.3.0"
  end
end
```
The latter might be better for long-term maintainability, as the tendency is to remove support for older versions of the framework over time as they become deprecated.

You'll need to update the `appraisal` files after that with:
```sh
$ appraisal clean
$ appraisal generate
```

Reply:
Thanks for the tips!